### DISCUSSION TITLE
Nighlty build (1.2.0) works only in share screen mode and messes up configuration

### DISCUSSION THREAD

**Discussion by pedjas** [rustdesk/rustdesk#2149] [General]
Just installed nightly build. When connected to remote client it just shares remote screen. Keyboard and mouse are not working on remote client except scrolling view.

There are no options to set view. It shows remote view in full resolution but as it does not fit view window it has to be scrolled. Basicaly, there is no other option available except to close window

Remote client is 1.1.9 if that matters.

Then I realized that it messed up configuration for local 1.1.9 client to. After I once run 1.2.0 and exit, passwords saved for accessing remote clients were reset and I had to reenter them again in 1.1.9. Also 1.1.9 now works only in screen view mode.

**Comment by rustdesk** [comment]
@fufesou it seems still they can not find the menubar.

**Comment by rustdesk** [comment]
> Keyboard and mouse are not working on remote client

Really?

**Comment by pedjas** [comment]
After I accessed remote computers the other way and restarted RestDesk clients there, I could again connect using 1.1.9 client. It seems 1.20. managed to dumb up remote clients somehow.

If 1.2.0 is not compatible with older clients it should refuse to connect and inform user that it needs upgrade.

**Reply by rustdesk** [reply]
> If 1.2.0 is not compatible with older clients it should refuse to connect and inform user that it needs upgrade.

@asur4s 1.2.0 can not control 1.1.9?

**Reply by chiehw** [reply]
<img width="1371" alt="image" src="https://user-images.githubusercontent.com/99897242/202124717-ea05d43e-2215-409c-b0ee-e8b8bfab71e6.png">
This is the result of my test just now.

**Reply by chiehw** [reply]
1.20 -> 1.19

**Comment by rustdesk** [comment]
> After I once run 1.2.0 and exit, passwords saved for accessing remote clients were reset and I had to reenter them again in 1.1.9. Also 1.1.9 now works only in screen view mode.

This is intended. something not compatible between 1.1.9 and 1.2.0.

**Reply by pedjas** [reply]
That is ok. It just should not even attempt connection as it dumbs up remote clients. 

Luckily I have alternative to connect to fix it, but if RustDesk was only way to access remote computers it would mean i had to go physically to remote computer.

**Comment by pedjas** [comment]
I repeated test and got the same results.

I run 1.2.0.

Connect to one client, it shows picture but it is full resolution. Mouse and keyboard do not work, except scrolling content as remote image is displayed in full resolution, not scalled to be visible in whole.

When I connect other client it just shows message "Connected, waiting for image" and that is all. i can only close connection.

Then I close 1.2.0 and start 1.9.0.

When I connect to first remote, it asks me to enter password again, and after I do that it shows remote screen but mouse and keyboard do not work.

When I connect to second remote, it also asks to enter password again, and then shows message "Connected, waiting for image". Al i can do is close connection.

Then I access remote computers other way, restart RustDesk client and then I can again connect to them properly using 1.1.9 client.

**Reply by rustdesk** [reply]
1.2.0 looks like a shit.

**Reply by chiehw** [reply]
What OS is on the control side?

**Reply by pedjas** [reply]
Is this rustdesk account official?

**Reply by pedjas** [reply]
> What OS is on the control side?

All three clients and server are on Windows 10.

**Reply by chiehw** [reply]
I am responsible for fixing the keyboard issue of rustdesk, you mentioned that the keyboard and mouse do not work, I would like to know more information

**Reply by chiehw** [reply]
OK, thanks, I'll try to reproduce it

**Reply by pedjas** [reply]
> OK, thanks, I'll try to reproduce it

I can let you access my computer to check if that could help.

**Comment by pedjas** [comment]
I use self hosted RustDesk server. I am accessing both remote clients from Internet. Both remote clients and server are in the network which is behind NAT. Maybe that has some influence to this issue?

When I use all 1.1.9 clients it works like a charm.

**Comment by pedjas** [comment]
I did another test. This time I run version 1.2.0 on two clients and connected them. 

This time mouse and keyboard worked but still remote screen is displayed in full resolution and as it does not fit viewing window I have to scroll it around. There is no menu and no options to set view or anything else.

**Reply by rustdesk** [reply]
Is it that hard to find the menu bar? could you find carefully?

**Reply by rustdesk** [reply]
We do need to redesign, you are not the first one can not find it.

**Reply by pedjas** [reply]
> Is it that hard to find the menu bar? could you find carefully?

No, I cannot find it. I clicked ewerywhere that seemed as clickable. 

Maybe you could post screenshot and show where to look?
